modules/loki-create-object-storage-secret-cli.adoc

```diff
@@ -21,11 +21,11 @@ To configure Loki object storage, you must create a secret. You can do this by u
 [source,terminal]
 ----
 $ oc create secret generic -n openshift-logging <your_secret_name> \
- --from-file=tls.key=<your_key_file>
- --from-file=tls.crt=<your_crt_file>
- --from-file=ca-bundle.crt=<your_bundle_file>
- --from-literal=username=<your_username>
- --from-literal=password=<your_password>
+  --from-file=tls.key=<your_key_file>
```

Comment:
the backslash (\) must be set at the end of a line .  as it is a line continuation character (or line join symbol).